tox.ini

```diff
@@ -2,7 +2,7 @@
 minversion=2.3.1
 envlist =
     py{27,35}-ansible22-{pylint,unit,flake8}
-    yamllint
+    py27-yamllint
```

Comment:
We could, but didn't know how much difference there would be between the two for yaml parsing.  I was unsure as to why you didn't just have it like this:

`py{27,35}-ansible22-{pylint,unit,flake8,yamllint}`